### DISCUSSION TITLE
Show hidden HTML entities like &shy; in Article Title

### DISCUSSION THREAD

**Discussion by motorentod** [joomla/joomla-cms#42375] [2. Cold Features]
I would like to see the entered &shy; html entities in the Article Titles or elsewhere in Joomla Backend. I think this might be just a problem for languages with large words that, like german or. Any clues how to solve this?

![example-of-article-title-with-shy-in-edit](https://github.com/joomla/joomla-cms/assets/63058321/189274a2-de05-41de-9a46-46ea08ce0f91)
![example-of-article-title-with-shy-in-list](https://github.com/joomla/joomla-cms/assets/63058321/5920a396-a7df-42d0-8445-4a0ab7a38b9d)
![example-from-yootheme-editor-for-shy](https://github.com/joomla/joomla-cms/assets/63058321/56e09459-49f7-4d58-a53a-08968b95304c)

**Comment by chmst** [comment]
I could imagine a plugin which replaces html entities.
If you write html entities as [&shy;] then the plugin can remove the brackets when it renders the menu in frontend and you see  it in backend.
